package.json

```diff
@@ -3,8 +3,7 @@
   "version": "1.0.2",
   "description": "Open Pixel is a JavaScript library for creating embeddable and intelligent tracking pixels",
   "main": "openpixel.min.js",
-  "dependencies": {},
-  "devDependencies": {
+  "dependencies": {
```

Comment:
Why switching this to `dependencies` instead of `devDependencies`. I just thought you wouldn't want it in `dependencies` because those may make it to the front end depending on your pipeline. But I guess this will be built separately probably. I don't know enough about it so just curious about your decision here.

Reply:
It depends on project setup. In my project I'm doing separate build for development, staging and production. So in my setup I have dev deps only for tests, and for building project I'm using prod deps, and this module I'm using as a build tool, not like a ready prebuilt code. And actually, prebuilt code that is in this project will not work, until you change configuration and run build by yourself. 

Other example of build things with env=production could be React for instance. To optimize performance of your frontend app for React you should use env=production, and in order to do so, probably you should keep all your frontend deps in deps, and not in devDeps. 
https://reactjs.org/docs/optimizing-performance.html

So, that are might thoughts.